weblab_cg/_printer.py

```diff
@@ -61,34 +61,35 @@ class Printer(sympy.printing.printer.Printer):
         A function that converts derivatives to strings.
 
     """
-    # List of functions handled by python's `math` module
+    # Dictionary of functions handled by python's `math` module and their corresponding names to be generated
     _math_functions = {
-        'acos',
-        'acosh',
-        'asin',
-        'asinh',
-        'atan',
-        'atan2',
-        'atanh',
-        'ceil',     # called ceiling in sympy
-        'cos',
-        'cosh',
-        'exp',
-        'expm1',
-        'factorial',
-        'floor',
-        'log',
-        'log10',
-        'log1p',
-        'log2',
-        'sin',
-        'sinh',
-        'tan',
-        'tanh',
+        'Abs': 'abs',
+        'acos': 'math.acos',
+        'acosh': 'math.acosh',
+        'asin': 'math.asin',
+        'asinh': 'math.asinh',
+        'atan': 'math.atan',
+        'atan2': 'math.atan2',
+        'atanh': 'math.atanh',
+        'ceiling': 'math.ceil',
+        'cos': 'math.cos',
+        'cosh': 'math.cosh',
+        'exp': 'math.exp',
+        'expm1': 'math.expm1',
+        'factorial': 'math.factorial',
+        'floor': 'math.floor',
+        'log': 'math.log',
+        'log10': 'math.log10',
+        'log1p': 'math.log1p',
+        'log2': 'math.log2',
+        'sin': 'math.sin',
+        'sinh': 'math.sinh',
+        'tan': 'math.tan',
+        'tanh': 'math.tanh',
     }
 
-    # List of custom defined functions we are allowed to output
-    _custom_functions = set()
+    # Dictionary of custom defined functions we are allowed to output and their corresponding names to be generated
```

Comment:
This is OK, but could also just add to _math_functions

Reply:
you didn't want that last time because `GetIntracellularAreaStimulus` isn't a math function :) but we could just call it functions

Reply:
Yeah with the prefixes gone that makes sense. Thanks!

Reply:
ok I also moved the default dict into the _weblab_printer, just to hopefully make more clear that we'd expect it to be different for each one. let me know what you think